### PR TITLE
Extend readme to include libXC installation

### LINKED ISSUE #1: Installation with conda
Just a few notes on what I found out while working with `conda` on this, installing dependencies and building is quite straight-forward with the currently available `libxc` package on conda-forge (if the version is selected correctly):

```
mamba create -n libxc-4 'libxc=4.*' -c conda-forge
mamba activate libxc-4
export PKG_CONFIG_PATH=$CONDA_PREFIX/lib/pkgconfig/:$PKG_CONFIG_PATH
cmake -B _build -G Ninja
ninja -C _build
```

There is another potentially compatible version of libxc in the psi4 channel, which provides the CMake module files, but no Fortran interface.

### LINKED ISSUE #56: Simple hints for libXC requirements
**What is your suggested feature? Please describe.**
More documentation for local compilation (particularly libXC)

**Is your feature request related to a problem? Please describe.**
Would be good to spell out:
* don't use debian libXC packages
* If using the libXM configure script, set a local install path (as the skprogs cmake needs it to be installed for the package files to work)

**Describe the solution you'd like**
Slight expansion of the README

### PR DESCRIPTION
**Pull request by vanderhe**
Eventually closes #56.

Closes #1.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -65,6 +65,39 @@ Follow the usual CMake build workflow:
...
+
+* Clone the official libXC repository and checkout the latest release tag, e.g.
+  ``6.2.2``::
+
+    git clone https://gitlab.com/libxc/libxc.git libxc
+    cd libxc/
+    git checkout 6.2.2
+
+* Configure the project, specify your compilers (e.g. ``gfortran`` and ``gcc``),
+  the install location (e.g. ``$HOME/opt/libxc``) and the build directory
```
Comment (bhourahine) [inline README.rst:81]: Probably should mention this needs to match the path above.

Comment (vanderhe) [reply README.rst:81]: Please see revised version.

File: README.rst [hunk 2]
```diff
@@ -46,7 +46,7 @@ Follow the usual CMake build workflow:
   CMake) or the ``PKG_CONFIG_PATH`` environment variable (if libXC was built
   with autotools) in order to guide the library search::
 
-    CMAKE_PREFIX_PATH=YOUR_LIBXC_INSTALL_FOLDER FC=gfortan cmake [...]
+    CMAKE_PREFIX_PATH=$HOME/opt/libxc FC=gfortan cmake [...]
```
[the remark this answers is not in the document]
Comment (vanderhe) [reply README.rst:49]: Would you agree with the revised version? I have now consistently used `YOUR_LIBXC_INSTALL_FOLDER` in both locations, so from my side this should be instructive enough.

[the remark this answers is not in the document]
Comment (vanderhe) [reply README.rst]: Examples for `YOUR_SKPROGS_INSTALL_FOLDER` and `YOUR_LIBXC_INSTALL_FOLDER` added.

**Review by bhourahine — APPROVED** [review]
Ready to merge

**Pull request merged**